src/pages/metapackages/venia-sample-data/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Venia Sample Data metapackage
+---
+# Venia Sample Data metapackage
+
+This metapackage contains Venia Sample Data modules you can install to help you develop your projects locally using sample data.
+
+## Installation as a git-based composer package
+
+1. Clone the magento2 git repository and check out the latest develop branch, e.g. 2.4-develop. You may also check out and use any 2.4 release tags.
+
+```
+git clone git@github.com:magento/magento2.git .
+```
+
+2. Create an ext directory in the root of the magento2 project directory:
```

Comment:
Edit this step to something like

Change directories to the vendor `magento2/ext`.